README.md

```diff
@@ -39,10 +39,29 @@ $ ./gradlew docker
 
 You can run the fat jar as follows.
 
+
+```console
+
+$ java -jar scalar-admin-<version>-all.jar -c <COMMAND> -a <IP1:Port1,IP2:Port2,...>
+```
+
+The command above can take effect on multiple ScalarDB or ScalarDL instances in one shot.
```

Comment:
```suggestion
```
Since we recommend using `Scalar Admin for Kubernetes`, describing `Scalar Admin` 's pros might be confusing. In other words, if we describe `Scalar Admin has pros, but we recommend Scalar Admin for Kubernetes`, it might cause a bit of confusion.

Also, of course `can take effect on multiple ScalarDB or ScalarDL instances in one shot` might be one of the pros, but I think it is a `side effect`. I think we didn't expect to pause multiple ScalarDB or ScalarDL instances when we implemented the `-a` option.

The main purpose of the `-a` option is `fixing the known issue of -s option`. In other words, we don't implement the `-a` option to pause multiple ScalarDB or ScalarDL instances. In other words, we haven't discussed `pausing multiple ScalarDB or ScalarDL instances` much.

So, I think it would be better to remove this sentence at this point.

(However, I think it's worth discussing the feature of `pausing multiple ScalarDB or ScalarDL instances`. And, I think it might be better to implement such a feature on Scalar Admin for Kubernetes side too. Let's discuss it later.)

Reply:
@kota2and3kan 

Ah, I got what you mean.

For now, it's not supposed to be used by the end users since we haven't thought about it carefully.

I will remove this sentence to avoid confusion.